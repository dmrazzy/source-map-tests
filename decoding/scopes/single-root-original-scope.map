{
  "version": 3,
  "file": "single-root-original-scope.js",
  "sources": [
    "original_0.js"
  ],
  "mappings": "",
  "names": [
    "global"
  ],
  "scopes": "BCAAA,CKA,ECAA,FK"
}