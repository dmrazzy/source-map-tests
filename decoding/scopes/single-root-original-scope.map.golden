{"file":"single-root-original-scope.js","mappings":[],"sources":[{"url":"original_0.js","content":null,"ignored":false,"scope":{"start":{"line":0,"column":0},"end":{"line":10,"column":0},"name":null,"kind":"global","isStackFrame":false,"variables":[],"children":[]}}],"ranges":[{"start":{"line":0,"column":0},"end":{"line":0,"column":10},"definitionIndex":0,"stackFrameType":"none","callSite":null,"bindings":[],"children":[]}]}